trip_app/trips/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from trips.models import TripRequest
+from trips.tests.trip_factory import TripRequestFactory
+
+
+@pytest.mark.django_db
+def test_approve():
+    """ Verify TripRequest model 'approve' method. """
+    trip_request = TripRequestFactory(status=TripRequest.ACTIVE)
```

Comment:
This could be a reusable fixture `def trip_request(): return TripRequestFactory(...)`

Reply:
Should be updated now.